python_ta/contracts/__init__.py

```diff
@@ -352,7 +352,16 @@ def _check_invariants(instance, klass: type, global_scope: dict) -> None:
             _debug(f"Warning: could not evaluate representation invariant: {invariant}")
         else:
             if not check:
-                raise PyTAContractError(f'Representation invariant "{invariant}" was violated')
+                curr_attributes = ", ".join(
+                    f"{k}: {_display_value(v)}" for k, v in vars(instance).items()
+                )
+
+                curr_attributes = "{" + curr_attributes + "}"
+
+                raise PyTAContractError(
+                    f'Representation invariant "{invariant}" was violated for'
```

Comment:
@yibinglu as I was reviewing this I realized we can improve the error message further by writing the class name here as well (as we do on line 348, except use `instance.__class.__name__` instead of `.__qualname__`). So the message would read something like `Person representation invariant "..." was violated for ...`.

Please make that change, and update the changelog entry to mention that we're adding both the class name and instance attribute values to the message.